README.md

```diff
@@ -147,9 +147,9 @@ export default Ember.ArrayController.extend({
 
   // binding the property on the paged array
   // to the query params on the controller
-  pageBinding: "content.page",
-  perPageBinding: "content.perPage",
-  totalPagesBinding: "content.totalPages",
+  page: Ember.computed.alias("content.page"),
+  perPage: Ember.computed.alias("content.perPage"),
+  totalPages: Ember.computed.alias("content.totalPages"),
```

Comment:
So the default value initialization is no longer needed? Because they have a comment that says those default value definitions are important:

```javascript
// set default values, can cause problems if left out
// if value matches default, it won't display in the URL
page: 1,
perPage: 10
```